### PR TITLE
[MOS-000] Add changelog

### PR DESCRIPTION
**Pull request by SP2FET**
Added changelog

**Description**

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible.
- [x] Has documentation updated

### CODE REVIEW

**Review by Maciej-Mudita — APPROVED** [review]

File: pure_changelog.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+* Fixed problem with alphabetical sorting of contacts that do not contain the last name.
+* Fixed problem of being able to edit a deleted contact.
+* Fixed issue with the text informing about the Mudita OS update that appears during onboarding.
+* Fixed problem with using call log when more than 900 accounts are loaded.
+* Fixed problem with missing call information from the private number in the notification.
+* Fixed issue with the inability to interrupt an incoming quick call.
+* Fixed translations when adding, editing and deleting contacts from the call log.
+* Fixed issue with restoring data on Pure using Mudita Center.
+* Fixed issue with FLAC file playback.
+* Fixed problem with Music Player application crashing when appearing on file with unknown formats.
```
Comment (Lefucjusz) [inline pure_changelog.md:143]: "Fixed problem with Music Player application crashing when trying to play file with unknown format." sounds better IMHO ;)

**Review by Lefucjusz — APPROVED** [review]

File: pure_changelog.md (hunk 1 above)
Comment (SP2FET) [reply pure_changelog.md]: fixed

File: pure_changelog.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+* Extended audio files format support.
+
+### Changed / Improved
+
+#### General:
+* Changed the default text input type from ABC mode to Abc mode.
+
+### Fixed
+
+* Fixed problem with missing gong at beginning and end of meditation process.
```
[the remark this answers is not in the document]
Comment (SP2FET) [reply pure_changelog.md:113]: fixed

File: pure_changelog.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+* Fixed problem of loading quotes when displayed as wallpaper.
+* Fixed issue with Pure screen refresh after the phone is inactive for a long time.
+* Fixed issue with the phone freezing when the battery is very low while charging.
+* Fixed issues with the phone connecting to the network after turning it on.
+* Fixed flashlight shutdown in critical low-power event.
+* Fixed issue with the alarm list not updating after adding a new alarm.
+* Fixed functionality of adding contacts from the message screen.
+* Fixed bug of incoming call notifications appearing during tethering.
+* Fixed problem with popup after plugging USB even if no SIM is present.
+* Fixed problem with Pure unlocks after a missed call from favorite contact.
```
[the remark this answers is not in the document]
Comment (SP2FET) [reply pure_changelog.md:70]: fixed

File: pure_changelog.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+* Fixed problem with phone’s reboots while listening .wav/mp3.
+* Fixed problem of loading quotes when displayed as wallpaper.
+* Fixed issue with Pure screen refresh after the phone is inactive for a long time.
+* Fixed issue with the phone freezing when the battery is very low while charging.
+* Fixed issues with the phone connecting to the network after turning it on.
+* Fixed flashlight shutdown in critical low-power event.
+* Fixed issue with the alarm list not updating after adding a new alarm.
+* Fixed functionality of adding contacts from the message screen.
+* Fixed bug of incoming call notifications appearing during tethering.
+* Fixed problem with popup after plugging USB even if no SIM is present.
```
[the remark this answers is not in the document]
Comment (SP2FET) [reply pure_changelog.md:69]: sure

[the remark this answers is not in the document]
Comment (SP2FET) [reply pure_changelog.md]: fixed

File: harmony_changelog.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+* Changed exit screen with instruction to turn on.
+* Update Mudita logo icon.
+* Updated fonts.
+* Updated the screen at the end of the Onboarding process.
+
+#### Meditation:
+
+* Updated texts.
+* The time of appearance of messages has been extended.
```
Comment (Lefucjusz) [inline harmony_changelog.md:81]: More compact version without two "of" - "Messages display time has been extended."

**Review by Lefucjusz — APPROVED** [review]

**Pull request merged**